src/modules/database/migrations/1674066164956-AddProductCategoryAndProductBaseTables.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddProductCategoryAndProductBaseTables1674066164956 implements MigrationInterface {
+    name = 'AddProductCategoryAndProductBaseTables1674066164956'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "product_category" ("id" SERIAL NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), "name" character varying NOT NULL, CONSTRAINT "UQ_96152d453aaea425b5afde3ae9f" UNIQUE ("name"), CONSTRAINT "PK_0dce9bc93c2d2c399982d04bef1" PRIMARY KEY ("id"))`);
```

Comment:
Would be great to format it somehow. For instance check migration AddUserTable. One field = one line